apps/wing/src/cli.ts

```diff
@@ -178,12 +128,18 @@ async function main() {
       void import("./commands/upgrade").then((m) => m.checkForUpdates());
     }
   }
+  function loadEnv(cmd: Command) {
+    loadEnvVariables({
+      mode: cmd.args[0],
```

Comment:
Do `wing it` and `wing run` load from distinct environment files even if it's the same mode?

Reply:
Good point, I oversimplified it

Reply:
I think you actually did a great job. Thanks to you, I could find a simpler way to load the env variables and make sure that also using `wing it` sets the right mode.